composer/trainer/trainer.py

```diff
@@ -1412,7 +1412,7 @@ def _accumulate_time_across_ranks(
     def _train_loop(self) -> None:
         """Run training for the specified number of epochs and log results."""
         # print training start
-        self.logger.data_fit({'trainer/algorithms': [str(algo) for algo in self.state.algorithms]})
+        self.logger.data_fit({algo.__class__.__name__: 1 for algo in self.state.algorithms})
```

Comment:
I presume this is because tensorboard unable to handle lists of strings?

Reply:
yes. plus this makes it more readable for any logger (it actually displays the class name and not the object name with memory address)